message LoginRequest {
	required string name = 1;
}

message LoginResponse {
	required bool success = 2;
}

message ChatRequest {
	required string message = 3;
}

message ChatNotify {
	required string name = 4;
	required string message = 5;
}